### PR TITLE
Add backup/restore caveat

### PR DESCRIPTION
**Pull request by guangyee**

### CODE REVIEW

**Review by jeremy-moffitt — APPROVED** [review]

File: content/rancher-prime/azure/contents.lr [hunk 1]
```diff
@@ -21,18 +21,22 @@ body:
...
+3. Click 'Create' 
 
 ### Basics
 
 1. Select existing subscription from the dropdown list.
 2. Select existing Resource group from the dropdown list.
 3. Select existing AKS Cluster Name from the dropdown list.
 4. Choose name for Cluster extension resource name. It can be consisted of
-   alphanumeric and dots, with the maximum lenght of 253 characters.
+   alphanumeric and dots, and the value must be between 2 and 253 characters.
```
Comment (mbelur) [inline content/rancher-prime/azure/contents.lr]: nit: s/value/length

File: content/rancher-prime/azure/contents.lr [hunk 2]
```diff
@@ -286,7 +290,13 @@ AKS cluster already has the extension. Uninstall the extension as it suggested i
 
 ## Known Issues
 
-1. Uninstalling Rancher Prime may not cleanly remove all the resources that were
+1. When migrating Rancher to a different AKS by following the steps specified in
+   [Rancher Backups and Disaster Recovery](https://ranchermanager.docs.rancher.com/pages-for-subheaders/backup-restore-and-disaster-recovery), Rancher Prime must be reinstalled via the Azure
+   marketplace on the targett AKS cluster after restoring from the backup.
```
Comment (mbelur) [inline content/rancher-prime/azure/contents.lr]: nit: s/targett/target

File: content/rancher-prime/azure/contents.lr (hunk 1 above)
Comment (guangyee) [reply content/rancher-prime/azure/contents.lr]: good catch!

File: content/rancher-prime/azure/contents.lr (hunk 2 above)
Comment (guangyee) [reply content/rancher-prime/azure/contents.lr]: good catch!

**Review by chajain — COMMENTED** [review]
Should we also specify not to install Rancher on target EKS cluster before the restore

**Comment by jeremy-moffitt** [comment]
> Should we also specify not to install Rancher on target EKS cluster before the restore

I think thats implied by 

> Rancher Prime must be reinstalled via the Azure marketplace on the target AKS cluster after restoring from the backup.

but happy to reconsider if you disagree

**Review by mbelur — APPROVED** [review]
LGTM

**Pull request merged**